superbit_lensing/ngmix_fit/ngmix_fit.py

```diff
@@ -11,25 +11,25 @@
 
 parser = ArgumentParser()
 
-parser.add_argument('--medsfile', type=str, default=None,
+parser.add_argument('medsfile', type=str,
                     help='MEDS filename to run ngmix on')
-parser.add_argument('--outfile', type=str, default=None,
+parser.add_argument('outfile', type=str,
                     help='Name of output file')
-parser.add_argument('--config', type=str,
+parser.add_argument('config', type=str,
                     help='ngmix config filename')
-parser.add_argument('--outdir', type=str, default=None,
+parser.add_argument('-outdir', type=str, default=None,
                     help='Output directory')
-parser.add_argument('--start', type=int, default=None,
+parser.add_argument('-start', type=int, default=None,
                     help='Starting index for MEDS processing')
-parser.add_argument('--end', type=int, default=None,
+parser.add_argument('-end', type=int, default=None,
                     help='Ending index for MEDS processing')
-parser.add_argument('--n', type=int, default=1,
+parser.add_argument('-n', type=int, default=1,
                     help='Number of CPU cores to use')
 parser.add_argument('--clobber', action='store_true', default=False,
                     help='Set to overwrite existing outfile')
 parser.add_argument('--test', action='store_true', default=False,
                     help='Use to do a test run w/ provided config')
-parser.add_argument('-v', action='store_true', default=False,
+parser.add_argument('--vb', action='store_true', default=False,
                     help='Verbose')
 
```

Comment:
It seems like this code does everything that `ngmix_fit_superbit3.py` does except for making diagnostic plots and calculating shear moments using the full responsivity matrix ("`gMC`"). Is the plan to start running `ngmix_fit` instead?

Reply:
That was the long term goal (as I'd like this to be `ngmix v2.X` compatible), but I don't think I ever showed that it worked for metacal fits

Reply:
btw I haven't touched this file in many months, I just updated it to work w/ the pipeline flag arg changes